### PR TITLE
Make container fluid to fill the pane

### PR DESCRIPTION
**Pull request by leigh-pointer**

### CODE REVIEW

**Comment by hishamco** [comment]
Again I see some unrelated changes, I'm not sure how you start your PR every time. Don't understand me wrong, I appreciate your contribution but the extra commits and unnecessary changes are annoying

**Comment by leigh-pointer** [comment]
These are two commits from me.

**Comment by hishamco** [comment]
Ok, but I saw some changes related to  the icon

**Comment by sbwalker** [comment]
@leigh-pointer click on the Files Changes tab above - you will see a bunch of additional files were included that are unrelated to the PR

File: Oqtane.Client/Themes/OqtaneTheme/Containers/Container.razor [hunk 1]
```diff
@@ -17,7 +17,7 @@
         <ModuleActions />
     }
     <div class="row px-4">
-        <div class="container">
+        <div class="container-fluid">
```
Comment (hishamco) [inline Oqtane.Client/Themes/OqtaneTheme/Containers/Container.razor:20]: Why we didn't use the field `_classes` here instead?

**Review by hishamco — CHANGES_REQUESTED** [review]

**Comment by hishamco** [comment]
Please squash the commits into a single one

**Pull request merged**